src/tools/stmo.md

```diff
@@ -133,50 +100,29 @@ ORDER BY
   mau DESC
 ```
 
-If you type that into the main query edit box and then click on the "Execute"
-button, you should see a blue bar appear below the edit box containing the text
-"Executing query..." followed by a timer indicating how long the query has been
-running. After a reasonable (for some definition of "reasonable", usually less
-than a minute) amount of time the query should complete, resulting in a
-table showing a MAU value for each country. Congratulations, you've
-just created and run your first STMO query!
-
-Now would be a good time to click on the large "New Query" text near the top of
-the page; it should turn into an edit box, allowing you to rename the
-query. For this exercise, you should use a unique prefix (such as your name)
-for your query name, so it will be easy to find your query later; I used
+If you type these parameters into the main query Edit field and then click the "Execute"
+button, a blue bar then appears below the Edit field. It includes the text "Executing query...", followed by a timer that indicates how long the query has run. After a reasonable period of time, usually less than a minute, the query typically completes its run. A table appears that displays a MAU value for each country. You have just created and run your first STMO query.
```

Comment:
I think this is one of those rare cases we can use an exclamation mark, since the user has accomplished something significant by this point in the tutorial.

```suggestion
If you type these parameters into the main query Edit field and then click the "Execute"
button, a blue bar then appears below the Edit field. It includes the text "Executing query...", followed by a timer that indicates how long the query has run. After some period of time, usually less than a minute, the query typically completes its run. A table appears that displays a MAU value for each country. You have just created and run your first STMO query!
```